### ISSUE TITLE
Model Training: Train the Sentiment Model

### ISSUE THREAD

**Issue by MeenaRhoshini** [MeenaRhoshini/Text-Classification-Sentiment-Analysis#3]
Implement the training loop using the training data, and set up loss function and optimizer for training the sentiment model in train.py.
